include/caffe/util/gpu_memory.hpp

```diff
@@ -0,0 +1,186 @@
+#ifndef CAFFE_UTIL_GPU_MEMORY_HPP_
+#define CAFFE_UTIL_GPU_MEMORY_HPP_
+
+#include <vector>
+#include "caffe/common.hpp"
+#ifndef CPU_ONLY
+
+namespace cub {
+  class CachingDeviceAllocator;
+}
+
+namespace caffe {
+
+struct GPUMemory {
+  static void GetInfo(size_t* free_mem, size_t* used_mem) {
+    return mgr_.GetInfo(free_mem, used_mem);
+  }
+
+  template <class Any>
+  static void allocate(Any** ptr, size_t size,
```

Comment:
Non-trivial methods should be CamelCase, following the style guide (e.g., `GetInfo` is above, so why the inconsistency here?).